lib/sorcery/providers/battlenet.rb

```diff
@@ -0,0 +1,53 @@
+module Sorcery
+  module Providers
+    # This class adds support for OAuth with BattleNet
+
+    class Battlenet < Base
+      include Protocols::Oauth2
+
+      attr_accessor :auth_path, :scope, :token_url, :user_info_path
+
+      def initialize
+        super
+
+        @scope          = 'openid'
+        @site           = 'https://eu.battle.net/'
```

Comment:
Will defaulting to the EU subdomain of battlenet cause issues for users in the US or other server areas?

Reply:
i actually forgot this, i planned to implement a option for this. but im not sure if it is critical maybee only some higher ping. i will implement the option anyway sor setting the location